docs/pages/build-reference/how-tos.md

```diff
@@ -60,15 +62,9 @@ This is an example of how your package.json might look like:
 ## How to use private package repositories
 
 - Configure your project in a way that works with `yarn` and relies on the `NPM_TOKEN` env variable to authenticate with private repositories
-- add `experimental.npmToken` in `credentials.json`
+- add `NPM_TOKEN` to your account or project's secrets. See the [secret environment variables](/build-reference/variables/#using-secrets-in-environment-variables) docs to learn how to do this.
 
-```json
-{
-  "experimental": {
-    "npmToken": "example npm token"
-  }
-}
-```
+<ImageSpotlight alt="Secret creation UI filled" src="/static/images/eas-build/environment-secrets/secrets-create-filled.png" />
 
 If you are not using `credentials.json` for Android/iOS credentials, it is fine for `experimental.npmToken` to be the only entry in the file. Add `credentials.json` to `.gitignore` if it's not there already.
```

Comment:
```suggestion
```

should remove this now that we don't expect anyone to put secrets in here, right?